skyportal/handlers/api/source.py

```diff
@@ -1138,6 +1143,11 @@ class Validator(Schema):
                         reverse=True,
                     )
 
+                if include_thumbnails and not remove_nested:
+                    obj_list[-1]["thumbnails"] = Thumbnail.query.filter(
```

Comment:
```suggestion
                    obj_list[-1]["thumbnails"] = Thumbnail.query_records_accessible_by(self.current_user).filter(
```

Reply:
The thumbnail read policy is just if the obj is readable, I believe, so I just bypassed the accessible part. But this is probably better anyways for future-proofing.

Reply:
Yeah, it's minor, but I heard some discussion about thumbnails not being read-public in the near future, so just wanted to be sure it's future-proofed